smart-campus-service-impl/src/main/java/hu/unideb/smartcampus/service/ejabberd/request/BaseRequest.java

```diff
@@ -0,0 +1,35 @@
+package hu.unideb.smartcampus.service.ejabberd.request;
+
+import java.io.Serializable;
+
+/**
+ * A parent class for every XMPP request.
+ *
+ */
+public class BaseRequest implements Serializable {
+
+  private static final long serialVersionUID = 1L;
+
+  /**
+   * The main host.
+   */
+  protected String host;
```

Comment:
Yes in this case you have to call the all arg constructor when initalieze the sub class. Super(host);